cpp/merkletree/compact_merkle_tree.h

```diff
@@ -20,16 +21,16 @@ class CompactMerkleTree : public cert_trans::MerkleTreeInterface {
  public:
   // The constructor takes a pointer to some concrete hash function
   // instantiation of the SerialHasher abstract class.
-  // Takes ownership of the hasher.
-  explicit CompactMerkleTree(SerialHasher* hasher);
-  CompactMerkleTree(const CompactMerkleTree& other, SerialHasher* hasher);
+  explicit CompactMerkleTree(std::unique_ptr<SerialHasher> hasher);
```

Comment:
Why did the "takes ownership" comment get removed from this constructor but not the other one?